air-backend/.env

```diff
@@ -0,0 +1,3 @@
+DGRAPH_VERSION=v21.03.1
+ECLIPSE_MOSQUITTO_VERSION=1.6.12
+LABS_AIR_VERSION=0.7.0
```

Comment:
I'm not sure, but I thought that trying to be consistent would be a good start. I also suspect that these files are only used in the folder where `docker-compose` is run (typically by the `start.sh` script). I'd like to be able to do a reasonable test of this, but I think it might require committing and allowing the release processing to assemble a distribution. 

While we use Makefile(s) to help assemble the products, there isn't a default setting or a documented command to run. I think the CI/CD knows this information, but it isn't accessible outside of the CI/CD configuration.